qutip/core/cy/cqobjevo.pyx

```diff
@@ -216,19 +219,22 @@ cdef class CQobjFunc(CQobjEvo):
     def call(self, double t, int data=0):
         return self.base(t, data=data)
```

Comment:
All these calls happen with Python semantics - is that intended?

Reply:
Could be made cpdef, but `self.base` is a python object so it probably have only a small impact.
And it won't survive the merger of QobjEvo and cQobjEvo.

Reply:
Yeah, the `self.base` was the bit I was asking about.  As long as it's expected that it's all Python semantics, that's fine.  I forgot which bits were meant to be fastcall and which weren't.